controller/controllers.js

```diff
@@ -49,4 +55,23 @@ const postComment = (request, response, next) =>
     })
 }
 
-module.exports = { getCategories, getReviews, getReviewsById, postComment };
\ No newline at end of file
+
+
+
+
+
+const getUsers = (request, response) =>
+{
+    fetchAllUsers()
+    .then((users) =>
+    {
+        response.status(200).send({'users': users});
```

Comment:
Should include a catch block that invokes next with any error

Reply:
added catch block.